pkg/crc/preflight/preflight_windows.go

```diff
@@ -120,7 +90,7 @@ var hypervPreflightChecks = []Check{
 		checkDescription: "Checking if the Hyper-V virtual switch exists",
 		check:            checkIfHyperVVirtualSwitchExists,
 		fixDescription:   "Unable to perform Hyper-V administrative commands. Please reboot your system and run 'crc setup' to complete the setup process",
-		flags:            NoFix,
+		flags:            NoFix | StartUpOnly,
```

Comment:
Wondering if `NoFix` + `fixDescription` should  be dropped from these tests?

Reply:
Yeah, dropped the flag and description